### PR TITLE
Add crud functionality

### PR DESCRIPTION
**Pull request by kumarsomeshunos**

### CODE REVIEW

File: src/routes/admin/Proposal.js [hunk 1]
```diff
@@ -5,16 +5,44 @@ import { verifyAdmin } from "../../middlewares/checkAuth";
...
 });
 
-
-router.delete("/delete/:id", verifyAdmin, async (req, res) => {
-
+router.get('/newproposal', verifyAdmin, async (req, res) => {
+    res.send('New proposal input field');
+})
+
+router.post('/newproposal', verifyAdmin, (req, res) => {
```
Comment (ayushpanwar25) [inline src/routes/admin/Proposal.js:22]: should admins be able to create new proposals?

File: src/routes/admin/Proposal.js [hunk 2]
```diff
@@ -5,16 +5,44 @@ import { verifyAdmin } from "../../middlewares/checkAuth";
...
 router.get("/get", verifyAdmin, async (req, res) => {
-
+    const proposals = await Proposal.find({});
+    res.send(proposals);
 });
 
 router.get("/get/:id", verifyAdmin, async (req, res) => {
-	
+	const _id = req.params.id;
+    const proposal = await Proposal.findById({ _id });
```
Comment (ayushpanwar25) [inline src/routes/admin/Proposal.js:14]: should be `findById(_id)`

File: src/routes/admin/Proposal.js (hunk 1 above)
Comment (jetchirag) [reply src/routes/admin/Proposal.js:22]:
```suggestion
router.post('/newproposal', verifyClub, (req, res) => {
```

Comment (ayushpanwar25) [reply src/routes/admin/Proposal.js:22]: this route goes in club/Proposal.js

**Pull request closed without merging**